tasks/sds.py

```diff
@@ -0,0 +1,34 @@
+import os
+import sys
+import tempfile
+from tasks.rtloader import get_dev_path
+
+from invoke import task
+
+is_windows = sys.platform == "win32"
+is_darwin = sys.platform == "darwin"
+
+@task
+def build_sds_library(ctx):
```

Comment:
nit: it would map better with other existing tasks to namespace it and rename the task to invoke it with `inv sds.build-library` or similar

Reply:
Should be addressed in latest commit.